en/ch01.md

```diff
@@ -25,9 +25,9 @@ You'll see a list of available languages, click on the one of your choice to cha
-(3) **Advanced mode:** In this mode your wallet will display all advanced features available, like changing fees/transaction priority, solo mining, message verification, transaction proof/check, offline transaction signing, key images importing, custom remote node (including .onion remote nodes), custom bootstrap node, starting a local node without setting a bootstrap node (maximum privacy), changing blockchain location, and daemon log. In this mode you can choose if you want to connect to the Monero network via a remote node or a local node. If you decide to run a local node, the blockchain file will be downloaded. If you want to start using Monero immediately, you can also set a bootstrap node (remote node) in the Settings > Node page. To achieve maximum privacy in your connection to the Monero network, it's recommended to avoid using a remote/bootstrap node and instead use advanced mode to start a local node without setting a bootstrap node.  
+(1) **Simple mode:** In this mode your wallet will display a simple interface with easy access to basic functionality like sending, receiving, transactions history, address book, accounts, and merchant mode, as well as hardware wallet support. It will **permanently connect to a third-party server** (a **remote node**), and it will not start a local node in your computer (the blockchain will not be downloaded). Choose this option if you don't have enough space to store the blockchain file, if you don't require maximum privacy and if you accept the privacy risks of using a remote node to connect to the Monero network.  
+(2) **Simple mode (bootstrap):** In this mode your wallet will also display a simple interface with easy access to basic functionality. Additionaly it will start a **local node** that synchronizes a **pruned blockchain** in the background. This means your local node will start downloading only a part of the blockchain file, and your blockchain will be [pruned](https://web.getmonero.org/resources/moneropedia/pruning.html) (1/3 of a full blockchain). This download can take several hours or even days to complete. While the pruned blockchain is being downloaded, your local node **will temporarily connect to a third-party server** (a **remote node** known as bootstrap node), which will allow you to start using Monero immediately. After downloading and synchronizing the pruned blockchain and the wallet, your local node will only temporarily connect to the bootstrap node when your blockchain is not updated. Be aware that using a remote node (including a bootstrap node) to connect to the Monero network may reduce your privacy.  
```

Comment:
The URL should always be `www.getmonero.org...` not `web.getmonero.org...`
```suggestion
(2) **Simple mode (bootstrap):** In this mode your wallet will also display a simple interface with easy access to basic functionality. Additionaly it will start a **local node** that synchronizes a **pruned blockchain** in the background. This means your local node will start downloading only a part of the blockchain file, and your blockchain will be [pruned](https://www.getmonero.org/resources/moneropedia/pruning.html) (1/3 of a full blockchain). This download can take several hours or even days to complete. While the pruned blockchain is being downloaded, your local node **will temporarily connect to a third-party server** (a **remote node** known as bootstrap node), which will allow you to start using Monero immediately. After downloading and synchronizing the pruned blockchain and the wallet, your local node will only temporarily connect to the bootstrap node when your blockchain is not updated. Be aware that using a remote node (including a bootstrap node) to connect to the Monero network may reduce your privacy.  
```